docs/quick-start.md

```diff
@@ -60,14 +36,17 @@ The first file, `admin/index.html`, is the entry point for the Netlify CMS admin
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <title>Content Manager</title>
 
-  <link rel="stylesheet" href="https://unpkg.com/netlify-cms@~0.4/dist/cms.css" />
+  <!-- Include the styles for the Netlify CMS UI, after your own styles -->
+  <link rel="stylesheet" href="https://unpkg.com/netlify-cms@>0.5.0-beta.1/dist/cms.css" />
 
 </head>
 <body>
-  <script src="https://unpkg.com/netlify-cms@~0.4/dist/cms.js"></script>
+  <!-- Include the script that builds the page and powers Netlify CMS -->
+  <script src="https://unpkg.com/netlify-cms@>0.5.0-beta.1/dist/cms.js"></script>
 </body>
 </html>
 ```
+*\*Note: The CMS links above pull from [unpkg](https://unpkg.com/#/), a service that serves [npm](https://www.npmjs.com/) packages over a CDN, with versioning built into the URL. In this example, we're pulling a beta version of the CMS (`@>0.5.0-beta.1`), in order to support Netlify Identity and Git Gateway. You will probably want to update these URLs after [version 0.5.0](https://github.com/netlify/netlify-cms/releases) is released.*
```

Comment:
`*\*`?

Reply:
The first `*` triggers the italics, and the second is escaped so it actually displays. I suppose it might be a little clearer if I use `_` for the italics.
(And, come to think of it, the visible asterisk isn't really appropriate here, because there's no referring asterisk. OK! I'll remove! :P )